mozci/console/commands/push.py

```diff
@@ -946,6 +955,107 @@ def list_classification_tasks(self, group_id):
             config.cache.add(cache_key, tasks, int(config["cache"]["retention"]))
 
 
+class CheckBackfillsCommand(Command):
+    """
+    Check if backfills on last pushes are finished and notify Sheriffs when they are.
```

Comment:
This docstring should be extended to list all the stages in the workflow:
- list NB pushes on branch
- check if they have any actions triggered by Treeherder, using TC index
- find potential backfill tasks
- check if all tasks in the group of a backfill tasks are completed
- send a notification
- index current task to avoid retrials